### ISSUE TITLE
Setting a global value for `maxQueryLimit` breaks `force:apex:execute`

### ISSUE THREAD

**Issue by surajp** [forcedotcom/salesforcedx-vscode#3455]
### Setting a global value for `maxQueryLimit` breaks `force:apex:execute`

When I set a value for `maxQueryLimit` to increase the number of records pulled by a soql query, `force:apex:execute` fails with `Value is not defined`

### Steps To Reproduce:

**Repository to reproduce:** [dreamhouse-lwc](https://github.com/dreamhouseapp/dreamhouse-lwc)

1.  Run `sfdx force:config:set --global maxQueryLimit=500000
2.  Create a `debug.apex` file with just one statement `System.debug('test')`
3.  Run the command `sfdx force:apex:execute -f debug.apex`

### Expected result

_The command should run successfully and we should see the debug statement in the console_.

### Actual result

_Command fails to run with this error message_ `ERROR running force:apex:execute:  Value is not defined`.

### System Information

Run `sfdx version --verbose --json` and paste the output here.

{
        "cliVersion": "sfdx-cli/7.110.0",
        "architecture": "wsl-x64",
        "nodeVersion": "node-v14.17.0",
        "pluginVersions": [
                "@oclif/plugin-autocomplete 0.3.0 (core)",
                "@oclif/plugin-commands 1.3.0 (core)",
                "@oclif/plugin-not-found 1.2.4 (core)",
                "@oclif/plugin-plugins 1.10.1 (core)",
                "@oclif/plugin-warn-if-update-available 1.7.0 (core)",
                "@oclif/plugin-which 1.0.3 (core)",
                "@salesforce/lwc-dev-server 2.10.0",
                "├─ @oclif/plugin-update 1.3.10",
                "└─ @oclif/plugin-help 2.2.3",
                "@salesforce/sfdx-diff 0.0.6",
                "@salesforce/sfdx-trust 3.6.0 (core)",
                "alias 1.1.10 (core)",
                "auth 1.7.1 (core)",
                "config 1.2.13 (core)",
                "evergreen 0.34.0",
                "└─ evergreen-build 0.20.2",
                "generator 1.1.8 (core)",
                "playground-export 1.2.0",
                "salesforcedx 51.13.0 (latest-rc)",
                "├─ apex 0.2.0",
                "├─ data 0.4.6",
                "├─ custom-metadata 1.0.12",
                "├─ limits 1.2.1",
                "├─ templates 51.4.0",
                "├─ user 1.2.11",
                "├─ @salesforce/sfdx-plugin-lwc-test 0.1.7",
                "├─ salesforce-alm 51.6.23",
                "├─ schema 1.0.7",
                "└─ org 1.6.6",
                "sfdmu 4.3.1",
                "sfdx-cli 7.110.0 (core)",
                "source 0.1.22",
                "telemetry 1.2.2 (core)"
        ],
        "osVersion": "Linux 5.4.72-microsoft-standard-WSL2"
}

**Comment by rcoringrato-sfdc** [comment]
@surajp Does the command execute properly when you just request help, e.g. `sfdx force:apex:execute -h`?

**Comment by surajp** [comment]
@rcoringrato-sfdc It seems to be working fine now. I have updated the cli. This is the current version info

```
{
        "cliVersion": "sfdx-cli/7.112.1",
        "architecture": "wsl-x64",
        "nodeVersion": "node-v14.17.2",
        "pluginVersions": [
                "@oclif/plugin-autocomplete 0.3.0",
                "@oclif/plugin-commands 1.3.0 (core)",
                "@oclif/plugin-help 3.2.2 (core)",
                "@oclif/plugin-not-found 1.2.4 (core)",
                "@oclif/plugin-plugins 1.10.1 (core)",
                "@oclif/plugin-update 1.4.0-3 (core)",
                "@oclif/plugin-warn-if-update-available 1.7.0 (core)",
                "@oclif/plugin-which 1.0.3 (core)",
                "@salesforce/sfdx-plugin-lwc-test 0.1.7 (core)",
                "@salesforce/sfdx-trust 3.6.0 (core)",
                "alias 1.1.10 (core)",
                "apex 0.2.3 (core)",
                "auth 1.7.1 (core)",
                "config 1.2.23 (core)",
                "custom-metadata 1.0.12 (core)",
                "data 0.6.0 (core)",
                "generator 1.1.8 (core)",
                "limits 1.2.1 (core)",
                "org 1.6.7 (core)",
                "salesforce-alm 52.2.3 (core)",
                "schema 1.0.8 (core)",
                "sfdx-cli 7.112.1 (core)",
                "source 1.0.7",
                "telemetry 1.2.3 (core)",
                "templates 52.1.0 (core)",
                "user 1.4.0 (core)"
        ],
        "osVersion": "Linux 4.19.104-microsoft-standard"
}
```
